praetorian_cli/ui/aegis/commands/job_helpers.py

```
@@ -351,3 +351,125 @@ def resolve_addomain_target_key(menu, domain):
+def select_asset_target(menu, default_hostname=None):
+    """Interactive asset/seed target picker with fuzzy search.
+
+    Returns:
+        tuple: (target_key, target_display) or (None, None) if cancelled.
+    """
+    colors = getattr(menu, 'colors', DEFAULT_COLORS)
+
+    try:
+        menu.console.print(f"  [{colors['dim']}]Loading seeds...[/{colors['dim']}]")
+        seeds, _ = menu.sdk.seeds.list()
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)